third_party/accessibility/ax/platform/ax_platform_node_win_unittest.cc

```diff
@@ -6752,7 +4532,7 @@ TEST_F(AXPlatformNodeWinTest, IValueProvider_SetValue) {
 
   EXPECT_UIA_ELEMENTNOTENABLED(provider1->SetValue(L"2"));
   EXPECT_HRESULT_SUCCEEDED(provider1->get_Value(bstr_value.Receive()));
-  EXPECT_STREQ(L"3", bstr_value.Get());
+  EXPECT_STREQ(L"3.000000", bstr_value.Get());
```

Comment:
Add a TODO here and link to https://github.com/flutter/flutter/issues/78460

Reply:
Done